github-sync/github-data/users.yaml

```diff
@@ -268,6 +268,12 @@ users:
         role: member
       - name: scaffolding-codeowners
         role: member
+  - username: kpk47
+    role: member
+    teams:
+      - name: public-good-instance-team
```

Comment:
Only need collaboator, you can drop this team (which gives triage)

Reply:
Done